server/mailet/integration-testing/src/test/java/org/apache/james/mailets/crypto/SMIMESignIntegrationTest.java

```diff
@@ -134,8 +128,7 @@ public void setup() throws Exception {
             .addProcessor(CommonProcessors.sieveManagerCheck())
             .build();
 
-        jamesServer = new TemporaryJamesServer(temporaryFolder, mailetContainer,
-            binder -> binder.bind(ZonedDateTimeProvider.class).toInstance(() -> DATE_2015));
+        jamesServer = new TemporaryJamesServer(temporaryFolder, mailetContainer);
```

Comment:
In fact I think the previous solution is more durable (havind a Date provider that returns a date where you are sure the certificate is valid). Can you use it everywhere (keeping the new certificate of course)